media-gfx/yafaray/yafaray-3.3.0.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6} )
+inherit cmake-utils python-r1 python-utils-r1
+
+DESCRIPTION="A free open-source montecarlo raytracing engine"
+HOMEPAGE="http://www.yafaray.org"
+SRC_URI="https://github.com/YafaRay/Core/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/YafaRay/Blender-Exporter/archive/v${PV}.tar.gz -> ${PN}-blender-exporter-${PV}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+blender debug +fastmath +fasttrig jpeg opencv openexr png +python -ruby tiff truetype"
+
+RDEPEND="dev-libs/boost
+	dev-libs/libxml2
+	sys-libs/zlib
+	blender? ( media-gfx/blender )
+	jpeg? ( virtual/jpeg:0 )
+	opencv? ( >=media-libs/opencv-3.1.0[openexr?] )
+	openexr? ( >=media-libs/openexr-2.2.0 )
+	png? ( media-libs/libpng:= )
+	python? ( ${PYTHON_DEPS} )
+	ruby? ( dev-lang/ruby:= )
+	tiff? ( media-libs/tiff:0 )
+	truetype? ( media-libs/freetype )"
+DEPEND="${RDEPEND}
+	dev-lang/swig"
+
+S="${WORKDIR}/Core-${PV}"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	sed -i "s/-Wall/${CXXFLAGS}/g" CMakeLists.txt || die # respect flags
+	sed -i "s/@YAFARAY_BLENDER_EXPORTER_VERSION@/v${PV}/" "${WORKDIR}/Blender-Exporter-${PV}/__init__.py" || die
+
+	pushd "${WORKDIR}/Blender-Exporter-${PV}"
+	eapply "${FILESDIR}/${P}-blender-exporter-paths.patch"
+	popd
+
+	eapply "${FILESDIR}/${P}-ruby-includes.patch"
+
+	eapply_user
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DYAF_LIB_DIR=$(get_libdir)
+		-DYAF_BINDINGS_RUBY_DIR="$(ruby -e 'print RbConfig::CONFIG["vendorarchdir"]')"
+		-DWITH_YAF_PY_BINDINGS=$(usex python)
+		-DWITH_YAF_RUBY_BINDINGS=$(usex ruby)
+		-DBLENDER_ADDON=OFF # addon is a separate package called blender-exporter
+		-DCMAKE_SKIP_RPATH=ON # NULL DT_RUNPATH security problem
+		-DDEBUG_BUILD=$(usex debug)
+		-DFAST_MATH=$(usex fastmath)
+		-DFAST_TRIG=$(usex fasttrig)
+		-DWITH_JPEG="$(usex jpeg)"
+		-DWITH_OpenCV="$(usex opencv)"
+		-DWITH_OpenEXR="$(usex openexr)"
+		-DWITH_PNG="$(usex png)"
+		-DWITH_QT=OFF # qt4 only at the moment
+		-DWITH_TIFF="$(usex tiff)"
+		-DWITH_Freetype="$(usex truetype)"
+	)
+
+	cmake-utils_src_configure
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	python_foreach_impl python_domodule "${BUILD_DIR}/src/bindings/yafaray_v3_interface.py"
+	python_foreach_impl python_domodule "${BUILD_DIR}/src/bindings/_yafaray_v3_interface.so"
```

Comment:
This is incorrect. You're installing module built for one version of Python to incompatible versions. Please use `python-single-r1` instead.